packages/grafana-ui/src/components/Combobox/Combobox.tsx

```diff
@@ -11,24 +11,23 @@ import { Input, Props as InputProps } from '../Input/Input';
 
 import { getComboboxStyles } from './getComboboxStyles';
 
-export type Value = string | number;
 export type Option = {
```

Comment:
```suggestion
export type SelectableOption = {
```
How about renaming this? I think this makes it clearer that it belongs to a select-like component. Also, do we want to put this in grafana/data?

Reply:
`ComboboxOption`?